trackmania.js

```diff
@@ -590,32 +716,32 @@ export class TrackmaniaFacade {
  * @returns {Promise<JSON>}
  */
 export async function embedTrackInfo(live_service, track_json) {
-
-    console.log(track_json);
+    const { command, title, author, authortime, goldtime, tags, website, stylename, thumbnail, mapUid, provision, mapType } = track_json;
 
     const medal_times = 
-        `:first_place: ${await live_service.getMapLeaderboard(`Personal_Best/map/${track_json.mapUid}`, 1).then(response => convertMS(response[0].score))}\n` +
-        `:green_circle: ${track_json.authortime}\n` +
-        `:yellow_circle: ${track_json.goldtime}`;
+        `:first_place: ${await live_service.getMapLeaderboard(`Personal_Best/map/${mapUid}`, 1).then(response => convertMS(response[0].score))}\n` +
```

Comment:
nit: Can we capture this in a variable to clean this up a little bit?

```js
const firstplacetime = await live_service.getMapLeaderboard(`Personal_Best/map/${mapUid}`, 1)
    .then(response => convertMS(response[0].score));
const medal_times = [
    `:first_place: ${firstplacetime}`,
    `:green_circle: ${authortime}`,
    `:yellow_circle: ${goldtime}`,
].join('\n');
```

Reply:
yes